### ISSUE TITLE
Rclone creating a duplicate folder on Amazon Cloud Drive

### ISSUE THREAD

**Issue by ayefly** [ncw/rclone#504]
When copying a folder to Amazon Cloud Drive, it occasionally creates a new folder with the same name, and uploads EVERYTHING in it to that new folder, instead of just new items to the old folder.

This results in a duplicate folder in amazon cloud.  
The existing folder is /RClone
The command I am using is rclone copy /mnt/user/pictures/RClone remote:/RClone
even the capitalization is the same on both folders. 
running rclone  size remote:RClone  showed the data for the new folder.
I then deleted the new folder, verified the old was still on the cloud drive, then ran size again.
rclone responded by saying couldn't find root
 
tried with remote:/RClone  as well as remote:RCLone
any ideas? i didnt even think amazon cloud allowed duplicates.

When filing an issue, please include the following information if
possible as well as a description of the problem.

> What is your rclone version (eg output from `rclone -V`)
1.29
> Which OS you are using and how many bits (eg Windows 7, 64 bit)
Unraid 6.1.9 - 64bit
> Which cloud storage system are you using? (eg Google Drive)
Amazon Cloud Drive
> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone copy /mnt/user/pictures/RClone remote:/RClone
> A log from the command with the `-v` flag (eg output from `rclone -v copy /tmp remote:tmp`)

rclone log Sat May 28 00:28:34 EDT 2016 

Starting rclone copy 
2016/05/28 00:28:35 Amazon cloud drive root 'RClone': Building file list
2016/05/28 00:28:35 Amazon cloud drive root 'RClone': Waiting for checks to finish
2016/05/28 00:29:35 
Transferred:     58965145 Bytes ( 952.14 kByte/s)
Errors:                 0
Checks:                 0
Transferred:           22
Elapsed time:      1m0.4s
Transferring:
 *                         153CANON/IMG_3985.JPG: 76% done. avg:  282.9, cur:  336.6 kByte/s. ETA: 1s
 *                         153CANON/IMG_4107.JPG: 61% done. avg:  266.0, cur:  335.8 kByte/s. ETA: 1s
 *                         Flip Video/raft.mp4:  4% done. avg:  251.0, cur:  244.0 kByte/s. ETA: 16m59s
 *           Ace Rafting - May 2013/IMG_1576.JPG: 55% done. avg:  278.7, cur:  284.1 kByte/s. ETA: 6s

**Comment by ncw** [comment]
That is very strange - ACD definitely doesn't allow duplicate files.

Can you see the folder if you do `rclone lsd remote:`?

Can you check in the web interface that it hasn't got a space on the end or something like that?

**Comment by ayefly** [comment]
I already deleted both RClone folders last night.  although I didn't run the lsd remote command, I did run rclone size remote:RClone.   when both folders were there it returned the size of the new folder.  when I deleted the new folder and ran the command again, it said directory not found.  so maybe I did have a space at the end of the original RClone folder.

I  created a new folder called test using ACD web interface, then ran a Cron job of RcloneCopy.sh on my pictures/RClone folder every hour.  worked using the "test" folder, copied the contents the first time, then recognized they were already there on subsequent runs.
